meteor/lib/collections/TranslationsBundles.ts

```diff
@@ -0,0 +1,17 @@
+import { TransformedCollection } from '../typings/meteor'
+import { registerCollection, ProtectedString } from '../lib'
+
+import { TranslationsBundle as BlueprintTranslationsBundle } from 'tv-automation-sofie-blueprints-integration'
+import { createMongoCollection } from './lib'
+
+/** A string identifying a translations bundle */
+export type TranslationsBundleId = ProtectedString<'TranslationsBundleId'>
+
+export interface TranslationsBundle extends BlueprintTranslationsBundle {
+	_id: TranslationsBundleId
```

Comment:
I'm missing a `blueprint: BlueprintId` here, so that a bundle is explicitly tied to the blueprint that created it.